mlc_llm/relax_model/param_manager.py

```diff
@@ -297,12 +313,21 @@ def set_param_loading_func(
         self.f_compute_relax_param = f_compute_relax_param
 
         self.model_path = model_path
+        self.use_safetensors = use_safetensors
+        if self.use_safetensors:
+            # Use a pointer here to prevent repeated import in tvm registered function
+            # pylint: disable=import-outside-toplevel
+            from safetensors.torch import load_file
+
+            self.safetensors_load_func = load_file
```

Comment:
Not sure if this is a good practice — using an attribute to point to an imported function. Was thinking that we do not want to make users install `safetensors` if they are not using it. 

The alternative I can think of is to import `load_file` in `load_torch_params_from_bin()`. Would this hurt the performance? Since it would ultimately be registered to tvm in the `get_item` function.

Reply:
Could we do some try-except testing when `--use-sefetensor` is true in `_parse_args` like this? https://github.com/mlc-ai/mlc-llm/blob/a856a4a1031ce6e588b7a15f95b94fb0dd74369d/mlc_llm/quantization/autogptq_quantization.py#L21-L26

Throwing exception early to indicate that safetensor is not importable.

Reply:
Added in the new commit, thanks!